conf/messages

```diff
@@ -493,6 +493,60 @@ otherBeneficiary.whenRemoved.error.invalid = Enter a real date when the benefici
 otherBeneficiary.whenRemoved.error.future = The date the beneficiary was removed from the trust must be today or in the past
 otherBeneficiary.whenRemoved.error.past = The date the beneficiary was removed must be on or after the date they were added
 
+otherBeneficiary.description.title = What is the description of the other type of beneficiary?
+otherBeneficiary.description.heading = What is the description of the other type of beneficiary?
+otherBeneficiary.description.checkYourAnswersLabel = What is the description of the other type of beneficiary?
```

Comment:
Prototype reads "What is the description of the beneficiary?". Check with Rachel if she wants it to mention "other" at all

Reply:
Checked and it is going to be "What is the description of the other beneficiary?" for everything